src/land/ascat_ssm2ioda.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python3
+
+#
+# (C) Copyright 2021 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+
+import sys
+import argparse
+import netCDF4 as nc
+import numpy as np
+from datetime import datetime, timedelta
+import os
+from pathlib import Path
+
+IODA_CONV_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
+if not IODA_CONV_PATH.is_dir():
+    IODA_CONV_PATH = Path(__file__).parent/'..'/'lib-python'
+sys.path.append(str(IODA_CONV_PATH.resolve()))
+
+import ioda_conv_ncio as iconv
+from collections import defaultdict, OrderedDict
+from orddicts import DefaultOrderedDict
+
+locationKeyList = [
+    ("latitude", "float"),
+    ("longitude", "float"),
+    ("datetime", "string")
+]
+
+obsvars = {
+    'soil_moisture': 'soilMoistureNormalized',
+}
+
+AttrData = {
+    'converter': os.path.basename(__file__),
+}
+
+
+class ascat(object):
+    def __init__(self, filename, mask, writer):
+        self.filename = filename
+        self.mask = mask
+        self.writer = writer
+        self.varDict = defaultdict(lambda: defaultdict(dict))
+        self.outdata = defaultdict(lambda: DefaultOrderedDict(OrderedDict))
+        self.loc_mdata = defaultdict(lambda: DefaultOrderedDict(OrderedDict))
+        self.var_mdata = defaultdict(lambda: DefaultOrderedDict(OrderedDict))
+        self.units = {}
+        self._read()
+
+    # Open input file and read relevant info
+    def _read(self):
+        # set up variable names for IODA
+        for iodavar in ['soilMoistureNormalized']:
+            self.varDict[iodavar]['valKey'] = iodavar, self.writer.OvalName()
+            self.varDict[iodavar]['errKey'] = iodavar, self.writer.OerrName()
+            self.varDict[iodavar]['qcKey'] = iodavar, self.writer.OqcName()
+            self.units[iodavar] = '%'
+        # open input file name
+        ncd = nc.Dataset(self.filename, 'r')
+        # set and get global attributes
+        AttrData["satellite"] = "METOP"
```

Comment:
METOP? Aren't there multiple? Like MetOp-A, MetOp-B,...?

Reply:
Yes, it will fit to Like MetOp-A/MetOp-B/MetOp-C. Do I need to put MetOp-A/B/C or something else? What is your suggestion?

Reply:
Not sure, I'll defer to any of the users of this dataset to see what they want. Doesn't matter to me.

Reply:
Okay, I checked my testinput nc file and its MetOp-A. I changed it as MetOp-A

Reply:
Then you don't need to hardcode this if it is in the input file?

Reply:
For user convenience,  I put AttrData["satellite"] = "METOp-A/METOp-B". This should be fine for users when looked at the "actual" input file together.  However, for testinput file (a subset example), it needs such an information.

Reply:
Cory, please wait for a minute. I will use ncd.getncattr('source') to get satellite such as satellite = "MetOp-A ASCAT" ; and I will removed sensor. This will avoid hard-code issue. I will test it and re-commit it.  You will review it later. Thank you.